hello-world-service/Ballerina.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+org = "wso2"
+name = "hello_world_service"
+version = "0.1.0"
+distribution = "2201.11.0"
+
+bi = true
```

Comment:
@hasithaa do we need this flag?

Reply:
I think we have to change the distribution version as well.

Reply:
We don't need this, untill we merge that, let's keep it.

Reply:
@LakshanWeerasinghe kept it ftm, cuz we still haven't migrated to the vscode setting